navigation/ca_cartographer/house_map/map.yaml

```diff
@@ -0,0 +1,6 @@
+image: map.pgm
```

Comment:
This file and `map.pgm` can be found [here](https://github.com/RoboticaUtnFrba/create_autonomy/tree/kinetic-devel/navigation/ca_move_base/maps).

Reply:
Removed this, because cartographer pure localization doesn't use maps, but pbstreams.